### DISCUSSION TITLE
[Feature] Photostream equivalent

### DISCUSSION THREAD

**Discussion by WildeTechSolutions** [immich-app/immich#8159] [Feature Request]
### Summary
- Photostream equivalent of iOS Photos app.
- Allow user accounts on the same server to subscribe to another users photostream, therefore receiving notifications when new photos are present.
- Owner of the photostream manually selects photos from his/her library to share to the stream.

### Background
Immich does a phenomenal job as a drop in Google Photos replacement, part is which is the ability to create and share albums.  However, much of family photo sharing that is done with iOS users is through photostreams which allows a user to post any new recent photo to the stream and everyone subscribing to the stream gets the notification that new photos have been posted.  

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Nice feature! I like this
